### PR TITLE
[5.1] Increase auth throttling from 3 fails to 5.

### PR DESCRIPTION
**Pull request by jesseleite**
I've typed in a wrong password 3 times before, often because of my stupid fingers and/or phone keyboard.  It would be annoying to see lockout message after only 3 failed attempted.  Increasing to 5 would be a bit more lenient on user, but shouldn't affect security.  If a password is really safe, it should take thousands of thousands of attempts, so increasing to 5 fails per minute shouldn't matter.

TL;DR: Punish the malicious bot, not the real user.

### CODE REVIEW

**Comment by JesseLeite** [comment]
Oh you know what... I think I liked where he's going in #9499 better.  Can I close this PR somehow?

**Comment by GrahamCampbell** [comment]
:+:

**Comment by GrahamCampbell** [comment]
:+1:

**Comment by JesseLeite** [comment]
Thanks @taylorotwell <3
And to go with this one: https://github.com/laravel/docs/pull/1650
